### ISSUE TITLE
Codeformer fails on MacOs

### ISSUE THREAD

**Issue by i3oc9i** [invoke-ai/InvokeAI#744]
When I ask for `codeformer` restoration, 

`dream> a photorealistic portrait of a nice cute girl -U 2 -G 0.8 -ft codeformer -save_orig`

I get the following error message 

```
>> CodeFormer - Restoring Faces for image seed:3239741084
>> Error running RealESRGAN or GFPGAN. Your image was not upscaled.
Placeholder storage has not been allocated on MPS device!
```

anyway the postprocessed image is coorectly scaled by `ERSGAN`, only `codeformer` is not executed.

FYI, if I remove the `-ft codeformer` option, then `GFPGAN` is correctly executed.

Also `codeformer` options are not recorded in the log.

The complete log
```
> python scripts/dream.py
* Initializing, be patient...

>> GFPGAN Initialized
>> CodeFormer Initialized
>> ESRGAN Initialized
>> Loading model from models/ldm/stable-diffusion-v1/model.ckpt
LatentDiffusion: Running in eps-prediction mode
DiffusionWrapper has 859.52 M params.
making attention of type 'vanilla' with 512 in_channels
Working with z of shape (1, 4, 32, 32) = 4096 dimensions.
making attention of type 'vanilla' with 512 in_channels
>> Using more accurate float32 precision
>> Model loaded in 5.52s
>> Setting Sampler to k_lms

* Initialization done! Awaiting your command (-h for help, 'q' to quit)
dream> a photorealistic portrait of a nice cute girl -U 2 -G 0.8 -ft codeformer -save_orig
/Users/ivano/Code/Ai/InvokeAi-dev/ldm/modules/embedding_manager.py:155: UserWarning: The operator 'aten::nonzero' is not currently supported on the MPS backend and will fall back to run on the CPU. This may have performance implications. (Triggered internally at  /Users/runner/work/_temp/anaconda/conda-bld/pytorch_1659484612588/work/aten/src/ATen/mps/MPSFallback.mm:11.)
  placeholder_idx = torch.where(
100%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 50/50 [00:20<00:00,  2.43it/s]
Generating: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 1/1 [00:21<00:00, 21.01s/it]
>> Real-ESRGAN Upscaling seed:3239741084 : scale:2x
	Tile 1/4
	Tile 2/4
	Tile 3/4
	Tile 4/4
>> CodeFormer - Restoring Faces for image seed:3239741084
>> Error running RealESRGAN or GFPGAN. Your image was not upscaled.
Placeholder storage has not been allocated on MPS device!
>> Usage stats:
>>   1 image(s) generated in 35.71s
Outputs:
[1] outputs/img-samples/000001.3239741084.png: "a photorealistic portrait of a nice cute girl" -s 50 -S 3239741084 -W 512 -H 512 -C 7.5 -A k_lms -G 0.8 -U 2.0
[2] outputs/img-samples/000001.3239741084.postprocessed.png: "a photorealistic portrait of a nice cute girl" -s 50 -S 3239741084 -W 512 -H 512 -C 7.5 -A k_lms -G 0.8 -U 2.0 0.75

dream> q
goodbye!
```

**Comment by BrentOzar** [comment]
This is a duplicate of #713. Before you open new issues in GitHub, do a quick search in that repo’s issues to see if someone already opened an issue for that status message. Cheers!

**Comment by i3oc9i** [comment]
closed in favor of #713
